docs/cookbook/creating-an-ft.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Creating a Fungible Token
+description: Clarity and Stacks.js code required to make a FT
+---
+
+Much the same as creating an NFT, Clarity also makes creating a fungible token (FT) trivial as well.
+
+The process and code are much the same.
+
+## Trait
+
+Just like with the NFT, it's a good idea to create a trait when you create a fungible token so that different tools and protocols and be confident in building interfaces for those tokens.
+
+Since FTs may be divisible, the [FT official mainnet deployment trait address](https://explorer.stacks.co/txid/0x99e01721e57adc2c24f7d371b9d302d581dba1d27250c7e25ea5f241af14c387?chain=mainnet) has additional functions.
```

Comment:
https://stacks-docs-lcj697nen-stacks-team.vercel.app/docs/cookbook/creating-an-ft
On the website, spelling error is there, but in the "Files Changed" codes, it is not there
But I'm gonna by the website anyways, needs addressing this error. 

Incorrect: Since FTs may be divisible, the [FT official mainnet deployment trait adress]
Correct: Since FTs may be divisible, the [FT official mainnet deployment trait address]